dataspaces.conf

```diff
@@ -2,10 +2,10 @@
 ## Config file for DataSpaces
 
 ndim = 2 
-dims = 600,600
+dims = 1600,1500
```

Comment:
@philip-davis - Could you elaborate a bit on what these mean and why you are modifying them? Thanks.